CONTRIBUTING.md

```diff
@@ -66,24 +70,30 @@ which is installed in editable mode, or you can use `python -m pdm` from inside
 $ pdm run test
 ```
 
-The test suite is still simple and requires to be supplied, please help write more test cases.
+The test suite is still simple and needs expansion! Please help write more test cases.
 
 ### Code style
 
-PDM uses `pre-commit` for linting, you need to install `pre-commit` first, then:
+PDM uses `pre-commit` for linting. Install `pre-commit` first, then:
 
 ```bash
 $ pre-commit install
 $ pdm run lint
 ```
 
-PDM uses `black` coding style and `isort` for sorting import statements, if you are not following them,
-the CI will fail, and your Pull Request will not be merged.
+PDM uses `black` for code style and `isort` for sorting import statements. If you are not following them,
+the CI will fail and your Pull Request will not be merged.
+
+### News fragments
 
+When you make changes such as fixing a bug or adding a feature, you must add a news fragment describing
+your change. News fragments are placed in the `news/` directory, and should be named according to this
+pattern: `<issue_num>.<issue_type>.md` (e.g., `566.bugfix.md`). The contents of the file should be a single sentence in the imperative
```

Comment:
Better to list all available issue types and their meanings, the information can be found at `pyproject.toml` under `tool.towncrier.type` table

Reply:
Done.